### DISCUSSION TITLE
Perte de débit vidéo conversion P7 to P8

### DISCUSSION THREAD

**Discussion by mutchu** [R3S3t9999/DoVi_Scripts#54] [General]
Bonjour,
Pourquoi j'ai une légère perte de débit vidéo sur un remux en le convertissant du profil 7 au profil 8 alors que ce n'est pas un film en FEL mais bien en MEL ?
Merci

**Comment by mutchu** [comment]
le débit perdu correspond au débit du Dolby vision, bizarre...

**Comment by R3S3t9999** [comment]
Car certain films ont une MEL de 2mbps au lieu du 68kbps habituel. 
Inquiète toi pas, si le script dit que c'est MEL, il n'y aura aucune perte dans la conversion P7 vers P8

**Reply by mutchu** [reply]
Merci pour ton retour et bravo pour ton travail ;)
Donc même si mediainfo m'indique une perte de débit par rapport au fichier original si le fichier est de base en MEL c'est pas grave...

**Reply by mutchu** [reply]
Avant la conversion :
Video
ID                                       : 1
ID in the original source medium         : 4113 (0x1011)
Format                                   : HEVC
Format/Info                              : High Efficiency Video Coding
Format profile                           : Main 10@L5.1@High
HDR format                               : Dolby Vision, Version 1.0, dvhe.07.06, BL+EL+RPU, Blu-ray compatible / SMPTE ST 2086, HDR10 compatible
Codec ID                                 : V_MPEGH/ISO/HEVC
Duration                                 : 2 h 56 min
Bit rate                                 : 53.1 Mb/s
Width                                    : 3 840 pixels
Height                                   : 2 160 pixels
Display aspect ratio                     : 16:9
Frame rate mode                          : Constant
Frame rate                               : 23.976 (24000/1001) FPS
Color space                              : YUV
Chroma subsampling                       : 4:2:0 (Type 2)
Bit depth                                : 10 bits
Bits/(Pixel*Frame)                       : 0.267
Stream size                              : 65.4 GiB (87%)
Default                                  : Yes
Forced                                   : No
Color range                              : Limited
Color primaries                          : BT.2020
Transfer characteristics                 : PQ
Matrix coefficients                      : BT.2020 non-constant
Mastering display color primaries        : Display P3
Mastering display luminance              : min: 0.0001 cd/m2, max: 1000 cd/m2
Maximum Content Light Level              : 413 cd/m2
Maximum Frame-Average Light Level        : 65 cd/m2
Original source medium                   : Blu-ray

**Reply by mutchu** [reply]
Après la conversion :
Vidéo
ID                                       : 1
Format                                   : HEVC
Format/Infos                             : High Efficiency Video Coding
Format, Profil                           : Main 10@L5.1@High
Format HDR                               : Dolby Vision, Version 1.0, Profile 8.1, dvhe.08.06, BL+RPU, no metadata compression, HDR10 compatible / SMPTE ST 2086, Version HDR10, HDR10 compatible
Identifiant du codec                     : V_MPEGH/ISO/HEVC
Durée                                    : 2 h 56 min
Débit                                    : 51,1 Mb/s
Largeur                                  : 3 840 pixels
Hauteur                                  : 2 160 pixels
Facteur de forme l/h                     : 16/9
Type de débit im/s                       : Débit im/s constant
Débit im/s                               : 23,976 (24000/1001) im/s
Espace de couleurs                       : YUV
Sous-échantillonnage de la chrominance   : 4:2:0 (Type 2)
Profondeur binaire                       : 10 bits
Bits/(Pixel*Image)                       : 0.257
Taille du flux                           : 62,8 Gio (87%)
Langue                                   : English (US)
Par défaut                               : Oui
Mode forcé                               : Non
Gamme de couleurs                        : Limited
Coordonnées de chromaticité              : BT.2020
Caractéristiques du transfert            : PQ
Coefficients de la matrice               : BT.2020 non-constant
Coordonnées de chromaticité de l'écran d : Display P3
Luminance de l'écran de référence        : min: 0.0001 cd/m2, max: 1000 cd/m2
Niveau de luminosité maximal du contenu  : 413 cd/m2
Niveau de luminosité moyen du contenu    : 65 cd/m2

**Reply by mutchu** [reply]
Le film c'est The Batman

**Comment by R3S3t9999** [comment]
oui c'est normal et ce film a une MEL de 2mbps ce qui correspond a la perte que tu vois dans ton mediainfo.
Peu importe le bitrate de MEL, il n'y aura pas de perte dans la conversion.

Disc Size:      97,479,349,961 bytes
Protection:     AACS2
Extras:         Ultra HD, BD-Java

PLAYLIST REPORT:

Name:                   00800.MPLS  
Length:                 2:56:11.602 (h:m:s.ms)
Size:                   96,647,104,512 bytes
Total Bitrate:          73.14 Mbps

(*) Indicates included stream hidden by this playlist.

VIDEO:

Codec                   Bitrate             Description     
-----                   -------             -----------     
MPEG-H HEVC Video       51013 kbps          2160p / 23.976 fps / 16:9 / Main 10 @ Level 5.1 @ High / 4:2:0 / 10 bits / 1000nits / HDR10 / BT.2020
MPEG-H HEVC Video     **2096 kbps (3.95%)**   1080p / 23.976 fps / 16:9 / Main 10 @ Level 5.1 @ High / 4:2:0 / 10 bits / 1000nits / Dolby Vision FEL / BT.2020

**Reply by mutchu** [reply]
Donc tu me confirme que tous les films MEL n'aurons aucune perte même si le débit change ?

**Comment by R3S3t9999** [comment]
oui

**Reply by mutchu** [reply]
Merci pour ton retour et encore bravo pour le taf
